tensorflow/compiler/mlir/lite/transforms/optimize_patterns.td

```diff
@@ -742,7 +742,7 @@ def ShapeMatchesReduceWithKeepAxes : Constraint<CPred<
 // Fold reshapes re-inserting reduced dimensions into the results of a reduction
 // with `keep_dims=false` by changing it to one using `keep_dims=true`.
 foreach ReduceOp = [TFL_ReduceMaxOp, TFL_ReduceMinOp, TFL_ReduceProdOp,
```

Comment:
Lets keep the ops sorted here (e.g., start with TFL_MeanOp).

Reply:
Done, Thank you!